app/controllers/gems_adoptions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class GemsAdoptionsController < ApplicationController
+  
+  def index
+    @gems_adoptions = GemsAdoption.all
+  end
```

Comment:
Agree with @sebasoga. I think this action is going to be implemented in the next planned task **List gems available for adoption** that @angela4713 started to implement this week. I think this action is out of the scope of this PR.